specification/securityandcompliance/resource-manager/Microsoft.SecurityAndCompliance/stable/2021-01-11/examples/ComplianceCenterOperationsList.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "api-version": "2021-01-11"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "Microsoft.SecurityAndCompliance/services/read",
+            "display": {},
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.SecurityAndCompliance/services/write",
+            "display": {},
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.SecurityAndCompliance/services/delete",
+            "display": {},
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.SecurityAndCompliance/locations/operationresults/read",
+            "display": {
+              "provider": "Microsoft.SecurityAndCompliance",
+              "resource": "operationresults",
+              "operation": "read",
+              "description": "Get the status of an asynchronous operation"
+            },
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.SecurityAndCompliance/checkNameAvailability/post",
```

Comment:
>post [](start = 75, length = 4)

/action, are you implementing this api?

Reply:
Deleted this api earlier. Will delete this from examples as well.